packages/fxa-auth-server/lib/routes/sign.js

```diff
@@ -9,10 +9,26 @@ const isA = require('joi');
 const validators = require('./validators');
 const SIGN_DOCS = require('../../docs/swagger/sign-api').default;
 const DESCRIPTION = require('../../docs/swagger/shared/descriptions').default;
+const crypto = require('crypto');
 
-module.exports = (log, signer, db, domain, devices) => {
+module.exports = (log, signer, db, domain, devices, config) => {
   const HOUR = 1000 * 60 * 60;
 
+  const rolloutRate = config.certificateSignDisableRolloutRate || 0;
+  // This determines if the certificate signing endpoint is disabled for a user.
+  // It is based on `BaseGroupingRule` from `fxa-content-server/app/scripts/lib/experiments/grouping-rules/base.js`
```

Comment:
thanks for the comment.  Good to know this is tested and you weren't just making it up :)